mimic/core.py

```diff
@@ -24,6 +24,66 @@
 from mimic.model.valkyrie_objects import ValkyrieStore
 
 
+class MimicCoreExceptions(Exception):
+    """
+    Parent for all Exceptions related to MimicCore
+    """
+    pass
```

Comment:
The "pass" is unnecessary, since the docstring fills the syntactic niche that would require it to be here.  Can you remove these?